nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -93,6 +93,8 @@ public abstract class AbstractConfig implements Config {
 
   protected Set<String> excludedClassAnnotations;
 
+  protected Set<String> generatedCodeAnnotations;
```

Comment:
At some point we should probably make these `ImmutableSet<String>` but not in this PR.